File: ABC/ABC227/c.c
#include <stdio.h>
int main(){
    long n = 0;
    long cnt = 0;
    long m = 0;
    long d = 0;

    scanf("%ld", &n);
    
    for(long i=1; i<n;i++){
        m = n / i;
        if (i*i*i > n){
            break;
        }
        for (long j=i; j<n;j++){
            d = m/j - j + 1;
            if (d <= 0){
                break;
            }
            cnt += d;
        }
    }

    if (n == 1){
        printf("1\n");
    }else{
        printf("%ld\n", cnt);
    }
}

File: ABC/ABC227/c.py
def main():
    n = int(input())
    cnt = 0

    for i in range(1, n):
        m = n//i
        if i**3 > n:
            break
        for j in range(i, n):
            d = m//j -j + 1
            if d <= 0:
                break
            cnt += d

    if n == 1:
        print(1)
    else:
        print(cnt)

if __name__ == "__main__":
    main()